programs/ssl/ssl_client2.c

```diff
@@ -631,6 +643,12 @@ int main( int argc, char *argv[] )
     mbedtls_net_context server_fd;
     io_ctx_t io_ctx;
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && \
+    defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED)
```

Comment:
~This is inconsistent with the guards used in `mbedtls_ssl_config`. We should be using the same guards everywhere.~ fixed in a later commit